jaydebeapi/__init__.py

```diff
@@ -398,7 +407,7 @@ def connect(jclassname, url, driver_args=None, jars=None, libs=None):
             libs = [ libs ]
     else:
         libs = []
-    jconn = _jdbc_connect(jclassname, url, driver_args, jars, libs)
+    jconn = _jdbc_connect(jclassname, url, driver_args, java_opts, jars, libs)
```

Comment:
This will fail on Jython. Please provide at least the parameter to _jdbc_connect_jython